src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1076,9 +1076,9 @@ private static void validatePropertySectionPropertyTokens(String fileName, Strin
         final String expectedDefaultTokenText = CheckUtil.getTokenText(check.getDefaultTokens(),
                 check.getRequiredTokens());
         if (expectedDefaultTokenText.isEmpty()) {
-            assertWithMessage("Empty tokens should have 'empty' string in xdoc")
-                .that(defaultTokenText)
-                .isEqualTo("empty");
+            assertWithMessage("Empty tokens should have '{@code \"\"}' string in xdoc")
+                    .that(defaultTokenText)
+                    .matches("\\{@code \"\"\\}|\\.|empty");
```

Comment:
This match operation is doing a way more than message say on mismatch.
Please put in message all possible values.

Why we are doing `|\\.` ? 

Please create separate  new issue to update all xdoc to have only `{@code ""}`. And put comment above as `until #xxxx`

Expected value in xdoc is `code>&quot;&quot;</code>` something is very broken in this error message.

Reply:
>Expected value in xdoc is `code>&quot;&quot;</code>` something is very broken in this error message.

Not addressed.
Xdoc is xml file. We print in message javadoc tag value.  We validate xdoc, so we should demand xml tags.

Reply:
Okay. Will try to address this. Meanwhile, i have opened an issue. Just wanted to know, is the issue relevant? I mean is there incosistency in representing empty values?

Reply:
In representation of empty values.

We have few options to match due to this inconsistency

Reply:
@Romani, have i understood it right? Basically, the error message should represent xml tags. 
XML Representation: `<code>&quot;&quot;</code>`
JavaDoc Tag Representation: `{@code ""}`
```
        if (expectedDefaultTokenText.isEmpty()) {
            assertWithMessage("Empty tokens should have" +
                    " '.', '<code>&quot;&quot;</code>','empty' string in xdoc")
                    .that(".".equals(defaultTokenText) || "empty".equals(defaultTokenText))
                    .isTrue();
        }
```

Reply:
Yes.

Reply:
As alternative we can change message
From 
`....string in xdoc`
To
`string in javadoc`